tests/e2e/specs/shopper/shopper-wc-blocks-checkout-failures.spec.js

```diff
@@ -0,0 +1,204 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+
+const { shopper, merchant, uiUnblocked } = require( '@woocommerce/e2e-utils' );
+
+/**
+ * Internal dependencies
+ */
+import {
+	shopperWCP,
+	merchantWCP,
+	describeif,
+	RUN_WC_BLOCKS_TESTS,
+} from '../../utils';
+
+const shippingDetails = config.get( 'addresses.customer.shipping' );
+const productName = config.get( 'products.simple.name' );
```

Comment:
I think it would be better if we leave the "import" area only for imports and move this assignments after the imports.

Reply:
I agree, but as mentioned in my previous comment above, only if we go for all the tests inside test suite.